.vscode/launch.json

```diff
@@ -130,11 +130,20 @@
       "request": "launch",
       "name": "Launch: import-events",
       "runtimeArgs": ["-r", "ts-node/register/transpile-only", "-r", "tsconfig-paths/register"],
-      "args": ["${workspaceFolder}/src/index.ts", "import-events", "--file", "/tmp/test-events-export.csv", "--wipe-db"],
+      "args": [
+        "${workspaceFolder}/src/index.ts", 
+        "import-events",
+        "--wipe-db",
+        "--force",
+        "--mode",
+        "preorg",
```

Comment:
Can you keep this one as a separate launch config? So we can still test archival separately